doc/advanced-configuration/database-backends-configuration.md

```diff
@@ -28,19 +28,19 @@ Transient data:
  Sooner or later a migration will be needed which may be painful.
  It is possible to store all data in Mnesia, but only for testing purposes, not for any serious deployments.
 
-* Redis - A fantastic choice for storing live data. 
- It's highly scalable and it can be easily shared by multiple MongooseIM nodes. 
- Additionally, Redis' great performance makes it an excellent choice for storing `user session` data. 
+* Redis - A fantastic choice for storing live data.
+ It's highly scalable and it can be easily shared by multiple MongooseIM nodes.
+ Additionally, Redis' great performance makes it an excellent choice for storing `user session` data.
  We recommend caution, since it has not yet been widely tested in production.
 
 
 Persistent Data:
 
-* RDBMS/ODBC - MongooseIM has a strong backend support for relational databases. 
- Considering the [CAP theorem](https://en.wikipedia.org/wiki/CAP_theorem) that usually guarantees both availability and consistency which is a great choice for regular MongooseIM use cases and features like `privacy lists`, `vcards`, `roster`, `private storage`, `last activity` and `message archive`. 
+* RDBMS/ODBC - MongooseIM has a strong backend support for relational databases.
+ Considering the [CAP theorem](https://en.wikipedia.org/wiki/CAP_theorem) that usually guarantees both availability and consistency which is a great choice for regular MongooseIM use cases and features like `privacy lists`, `vcards`, `roster`, `private storage`, `last activity` and `message archive`.
```

Comment:
something is wrong with this sentence


Considering the [CAP theorem](https://en.wikipedia.org/wiki/CAP_theorem) that usually guarantees both availability and consistency it is a great choice for regular MongooseIM use cases and features like `privacy lists`, `vcards`, `roster`, `private storage`, `last activity` and `message archive`?